packages/breadcrumbs/package.json

```diff
@@ -19,12 +19,11 @@
   "publishConfig": {
     "registry": "https://artifactory.availity.com/artifactory/api/npm/local-npm/"
   },
-  "dependencies": {
-    "prop-types": "^15.5.8"
-  },
   "devDependencies": {
+    "prop-types": "^15.5.8",
```

Comment:
For Travis to build the project you will also have to commit the package-lock file or else it will throw an error.

Reply:
right on! I like those changes you suggested! thanks!

Reply:
hey guys, it looks like every package-lock.json in the project was updated. Do i need to include all of those? or just the package-lock.json's next to the package.json's i modified?